include/fmt/base.h

```diff
@@ -427,6 +427,33 @@ enum class uint128_opt {};
 template <typename T> auto convert_for_visit(T) -> monostate { return {}; }
 #endif
 
+#ifndef FMT_USE_BITINT
+#  if FMT_CLANG_VERSION >= 1400
+#    define FMT_USE_BITINT 1
+#  else
+#    define FMT_USE_BITINT 0
+#  endif
+#endif
+
+template <class T, int N = 0> struct bitint_traits : std::false_type {};
```

Comment:
I don't think this should inherit from `false_type`. Instead you might need to add `is_formattable = false` to disable overloads for types other than `_BitInt`.